R/download_zenodo.R

```diff
@@ -144,3 +158,41 @@ download_zenodo <- function(doi,
   }
 }
 
+
+
+#' Human-readable binary file size
+#'
+#' Takes an integer (referring to number of bytes) and returns an optimally
+#' human-readable
+#' \href{https://en.wikipedia.org/wiki/Binary_prefix}{binary-prefixed}
+#' byte size (KiB, MiB, GiB, TiB, PiB, EiB)
+#'
+#' @param x An integer, i.e. the number of bytes (B).
+#'
+#' @return A string.
+#' @importFrom assertthat
+#' assert_that
+#' is.number
+#' @importFrom dplyr
+#' %>%
+#' @keywords internal
+human_filesize <- function(x) {
```

Comment:
Wouldn't it be useful to vectorise this function?

Reply:
Well, thought about it, but didn't do it - for now it's just an internal function with a single purpose. Would you export it? If so, then it's more worth it to broaden its use.

Reply:
Vectorised in bb248cd
Exported in 2f622fb